src/project/qml/MuseScore/Project/internal/NewScore/KeySignatureListView.qml

```diff
@@ -76,7 +76,13 @@ GridView {
             navigation.panel: root.navigationPanel
             navigation.row: root.columns === 0 ? 0 : Math.floor(model.index / root.columns)
             navigation.column: model.index - (navigation.row * root.columns)
-            navigation.accessible.name: keySignature.text
+            navigation.accessible.name: {
+                if (isSelected) {
+                    return keySignature.text;
+                } else {
+                    return keySignature.text + " " + qsTrc("project/newscore", "Not selected");
```

Comment:
Here you're asking the screen reader to say "not selected", but that's [not the right way](https://github.com/musescore/MuseScore/wiki/Accessibility-testing#controlling-speech-output) to solve this problem.

Instead, you need to ensure that the accessible object returns `true` for it's [selection state](https://doc.qt.io/qt-5/qaccessible-state.html), then leave the screen reader to decide what to do with that information.

If the screen reader still doesn't say "selected" or "not selected" then you need to change object's [accessible role](https://doc.qt.io/qt-5/qaccessible.html#Role-enum) until it does. The `ListItem` or `RadioButton` role should work here.

Reply:
At the moment, all this is implemented and the selected state is provided for accessibility, the problem is that this does not work.

Reply:
It's probably the [role](https://doc.qt.io/qt-5/qaccessible.html#Role-enum) that needs to be changed then, because the selection state is announced properly for the "Major" and "Minor" tabs at the top of this popup, and also for the note duration options in the Tempo popup.